src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerDataSource.java

```diff
@@ -143,18 +152,20 @@
     boolean getTransparentNetworkIPResolution();
 
     /**
-     * Sets a Boolean value that indicates if the trustServerCertificate property is enabled.
+     * Sets a value that indicates if the trustServerCertificate property is enabled.
```

Comment:
Why remove "boolean"? This method is still setting a boolean value no?

Reply:
you are right this WAS going to be changed but we decided against it